pybamm/expression_tree/concatenations.py

```diff
@@ -114,6 +115,25 @@ def is_constant(self):
         """See :meth:`pybamm.Symbol.is_constant()`."""
         return all(child.is_constant() for child in self.children)
 
+    def _sympy_operator(self, *children):
+        """Apply appropriate SymPy operators."""
+        concat_str = r"\\".join(map(sympy.latex, children))
+        concat_sym = sympy.Symbol(r"\begin{cases}" + concat_str + r"\end{cases}")
```

Comment:
these should also have indications of what domain each case holds in. Can be a separate PR
<img width="292" alt="Screen Shot 2021-07-13 at 1 15 03 PM" src="https://user-images.githubusercontent.com/20817509/125496624-5fb1f372-97c5-4e96-84e2-7e1eda641c37.png">